document/src/main/java/com/ritense/document/domain/impl/listener/ApplicationReadyEventListenerImpl.java

```diff
@@ -28,7 +28,7 @@ public ApplicationReadyEventListenerImpl(DocumentDefinitionService documentDefin
     }
 
     public void handle() {
-        documentDefinitionService.deployAll();
+        documentDefinitionService.deployAll(true, true);
```

Comment:
Ga je nu niet geforceerd document definitions upgraden zonder de gevolgen te weten voor bestaande documenten?

Reply:
Degene die de document definitie heeft aangepast in de /config/document/definition is daarvoor verantwoordelijk en hoort er dus rekening mee te houden wat dat doet voor bestaande documenten.

Reply:
Good to know, in short readonly concept stays in place for users that try to edit via de console.